patches/node/fix_tolocalstring_unicode_mismatch.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Fri, 25 Nov 2022 10:18:24 +0100
+Subject: fix: toLocalString Unicode mismatch
+
+Fixes a failure resultant of V8 updating to ICU 72.
+
+Refs https://chromium-review.googlesource.com/c/v8/v8/+/4004606.
+
+Date.toLocalString now uses uU+202f (narrow nbsp) before "PM" and "AM".
+
+This patch can be removed when https://github.com/nodejs/node/pull/45068
+is backported to Node.js v18.
+
+diff --git a/test/parallel/test-intl.js b/test/parallel/test-intl.js
+index 7d1742f2c7d1c6e335d3ccfc81ce0834c1de9c19..37d9749c684c4fe12f9c06d8bee9a16ea15bb708 100644
+--- a/test/parallel/test-intl.js
++++ b/test/parallel/test-intl.js
+@@ -97,7 +97,8 @@ if (!common.hasIntl) {
+   // Test format
+   {
+     const localeString = date0.toLocaleString(['en'], optsGMT);
+-    assert.strictEqual(localeString, '1/1/1970, 12:00:00 AM');
++    console.log(localeString)
```

Comment:
I'm assuming we don't need this console.log @codebytere?

Reply:
oh oops yes - can be deleted!